product_docs/docs/biganimal/release/administering_cluster/aws_secrets_manager_integration.mdx

```diff
@@ -0,0 +1,178 @@
+
+   Create the secret manager using a Lambda script or using the AWS console:
+
+   - Lambda script:
+
+     ```shell
+     import boto3
+     import json
+
+     def create_secret(secret_name, username, password, database, host):
+                client = boto3.client('secretsmanager')
+
+                secret_string = json.dumps({
+           "username": username,
+           "password": password,
+           "engine": "mysql",
+           "host": host,
+           "dbname": database,
+           "port": 3306
+         })
+
+    response = client.create_secret(
+        Name=secret_name,
+        SecretString=secret_string
+    )
+
+    return response
+    ```
+    
+    Using the created secret:
+
+    ```shell
+    create_secret('mySecretName', 'myUsername', 'myPassword', 'myDatabase', 'myHost')
+    ```
+
+   - AWS console:
+   
+     1. Search for Secret Manager under services.
+     1. Select **Store a new secret**.
+     1. Select **Credentials for other databases** on **Choose secret type** page and provide:
+        - Username
+        - Password
+        - Encryption key
+        - Database
+          Provide the server address, database name and port as per the selected database engine. Select **Next**.
+     1. Provide **Secret name** on the **Configure secret** page. Optionally you can provide:
+        - Description
+        - Tags
+        - Resource permissions
+        - Replicate secret
+        Select **Next**.
+     1. Optionally, provide details on the **Configure rotation** page.
+     1. Review the code in different languages like: Java, JavaScript, C#, Python3, Ruby, Go and Rust. Select **Store** to create the secret manager.
+
+1. Create secret in centralized Secrets Manager for your Access key.
+
+1. Create a sample login application:
+
+   For example using Lambda script,
+
+   ```shell
+   [cloudshell-user@ip-10-130-83-78 ~]$ cat lambda_connect.py 
+   import json
+   import boto3
+   import base64
+   import psycopg2
+   region = 'us-east-1'
+   
+   client = boto3.client('secretsmanager', region_name=region)
+   response = client.get_secret_value(
+               SecretId='dev/toy/demo'
+                   )
+   
+   secretDict = json.loads(response['SecretString'])
+   
+   connection = psycopg2.connect(
+            user=secretDict['username'], 
+             password=secretDict['password'],
+              host=secretDict['host'], 
+                port=secretDict['port'],
+                sslmode='require',
+                 database=secretDict['dbname'])
+   
+   mycursor = connection.cursor()
+   
+   create = "create table Demo0503(Toyota int)"
+   #sql = "INSERT into secretmgr(id,name) values (%s, %s)"
+   #value = (2, "Toyota_Demo")
+   mycursor.execute(create)
+   
+   connection.commit()
+   ```
+
+  Fetch all the rows from database:
+
+   ```shell
+   print(mycursor.rowcount, "record")
+   ```
+
+1. Create Secrets Manager rotation lambda function.
```

Comment:
```suggestion
1. Create a Secrets Manager rotation Lambda function.
```
I think Lambda should be capitalized as it is elsewhere? This is the name of a programming language?